spark-testing/spark_utils.py

```diff
@@ -129,6 +137,9 @@ def submit_job(
 
     conf_args += ['--conf', 'spark.mesos.role={}'.format(driver_role)]
 
+    if SPARK_DOCKER_USER is not None:
+        conf_args += ['--conf', 'spark.mesos.executor.docker.parameters=user={}'.format(SPARK_DOCKER_USER)]
```

Comment:
I noticed this line isn't being exercised by the integration tests. I'm just curious, do you know if some sort of quoting is required here due to the necessity of `=` characters in the value for `spark.mesos.executor.docker.parameters`? e.g. `spark.mesos.executor.docker.parameters="user=root"`

Reply:
this is not required although it's not quite reflected in the [docs](https://spark.apache.org/docs/2.2.0/running-on-mesos.html) but the source code here: https://github.com/mesosphere/spark/blob/master/resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackendUtil.scala#L150-L153 and here: https://github.com/mesosphere/spark/blob/master/resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackendUtil.scala#L111-L127 proves that quotes are not needed.

Reply:
Sweet!